src/FileType_HDF5.f90

```diff
@@ -971,6 +971,10 @@ SUBROUTINE mkdir_HDF5FileType(thisHDF5File,path)
     ! Convert the path to use slashes
     path2=convertPath(path)
 
+    path3=path2%substr(1,INDEX(path2,'/',.TRUE.)-1)
+    IF(.NOT.thisHDF5File%pathExists(CHAR(path3))) THEN
+      CALL thisHDF5File%mkdir(CHAR(path3))
```

Comment:
![image](https://user-images.githubusercontent.com/15932146/84340065-d285f200-ab6d-11ea-9c18-b72aa9afa929.png)

yeah, this is nice and slick